brontology/graph/entity/graph.py

```diff
@@ -12,8 +12,9 @@ def __init__(self):
     def add_token_relation(self, relation: TokenRelation):
         """Isolates the relation from the `TokenRelation` and adds it to this graph."""
 
-        self.add_link_by_member(
+        link = self.add_link_by_member(
             Lemma.from_token(relation.tail),
             Lemma.from_token(relation.predicate),
             Lemma.from_token(relation.head),
         )
+        link.sources.append(relation.source)
```

Comment:
Da die Relations jetzt einen `Excerpt` brauchen, wird der hier im Nachgang angehängt.
Im kern des Codes (`add_link_by_member`) geht das noch nicht, weil da die `Relation` in dieser Form noch nicht existiert.